package.json

```diff
@@ -9,15 +9,17 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run build-lib && npm run build-types",
+    "build": "npm run clean && npm run build-version && npm run build-lib && npm run build-types",
     "build-lib": "babel src -d lib --source-maps --extensions '.ts,.js'",
     "build-types": "tsc --emitDeclarationOnly",
-    "postversion": "git push origin master --follow-tags",
-    "prepublishOnly": "npm run build",
+    "build-version": "./timestamp.sh > src/versionInfo.ts  && eslint 'src/versionInfo.ts' --fix",
+    "watch": "babel src --out-dir lib --watch --source-maps --extensions '.ts,.js'",
```

Comment:
I think you might also want to run `timestamp.sh` in the `watch` script, otherwise the `import` in `/src/index.ts` might fail.